el-Joft/buyCoins
# Setting up the project and folder structure

#### What does this PR do?
- Setup the project and folder structure
#### Description of Task to be completed?
- Installed Django, Graphene
- Structured the project and created necessary folders
#### How should this be manually tested?
- clone this repo
- check out to `ft-project-setup`
- run `pip install -r requirements.txt`
- run `python manage.py runserver` to start the application
#### Any background context you want to provide?
- If Database is needed
- run `python manage.py migrate`
